lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -647,6 +647,11 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 		}
 
 		if ( in_array( 'styles', $types, true ) ) {
+			$root_block_key = array_search( static::ROOT_BLOCK_SELECTOR, array_column( $style_nodes, 'selector' ) );
+
+			if ( ! empty( $root_block_key ) ) {
```

Comment:
This needs to be a not `false` check, because `0` is the desired value as the root node is typically the first element ([array_search](https://www.php.net/manual/en/function.array-search.php) returns `false` when there is no match).

Reply:
Updated in: https://github.com/WordPress/gutenberg/pull/43792/commits/4720b61896e2210ddc59b11a6b401f239dfc4e65